### PR TITLE
Feat/explore details search

### PR DESCRIPTION
**Pull request by asogwachiuba**
Contains:
Places details feature
Place Filter
Explore places Home page

### CODE REVIEW

File: lib/features/events/views/explore/explore_events_view.dart [hunk 1]
```diff
@@ -54,7 +54,7 @@ class ExploreEventsView extends StackedView<ExploreEventViewModel> {
                               SizedBox(height: 30.h),
                               Visibility(
                                 visible: viewModel.exploreEventData
-                                        ?.featuredEvents?.isNotEmpty ??
+                                        ?.recommendedForYou?.isNotEmpty ??
```
Comment (divincoder) [inline lib/features/events/views/explore/explore_events_view.dart:57]: Whats going on here. You changed this from featured to recommended.

you can revert the changes if it was done to test

File: lib/features/events/views/explore/explore_events_view.dart [hunk 2]
```diff
@@ -64,32 +64,33 @@ class ExploreEventsView extends StackedView<ExploreEventViewModel> {
                               ),
                               Visibility(
                                   visible: viewModel.exploreEventData
-                                          ?.featuredEvents?.isNotEmpty ??
+                                          ?.recommendedForYou?.isNotEmpty ??
```
Comment (divincoder) [inline lib/features/events/views/explore/explore_events_view.dart:67]: Same here

File: lib/features/events/views/explore/explore_events_view.dart [hunk 3]
```diff
@@ -64,32 +64,33 @@ class ExploreEventsView extends StackedView<ExploreEventViewModel> {
...
                                       false,
                                   child: SizedBox(height: 10.h)),
                               Visibility(
                                 visible: viewModel.exploreEventData
-                                        ?.featuredEvents?.isNotEmpty ??
+                                        ?.recommendedForYou?.isNotEmpty ??
```
Comment (divincoder) [inline lib/features/events/views/explore/explore_events_view.dart:72]: Same here

File: lib/features/places/views/explore_places/explore_places_view.dart [hunk 4]
```diff
@@ -1,113 +1,22 @@
...
 import 'package:stacked/stacked.dart';
 
+import '../../../../core/widgets/ib_loader.dart';
+import '../../../../gen/assets.gen.dart';
+
 class ExplorePlacesView extends ViewModelBuilderWidget<ExplorePlacesViewModel> {
   @override
   Widget builder(
       BuildContext context, ExplorePlacesViewModel viewModel, Widget? child) {
-    // return Scaffold(
```
Comment (divincoder) [inline lib/features/places/views/explore_places/explore_places_view.dart:18]: If this is no longer useful then delete the commented code

**Review by divincoder — APPROVED** [review]

File: lib/core/constants/methods.dart [hunk 5]
```diff
@@ -671,7 +672,13 @@ class Methods {
   }
 
   static openSearchFilter(
-      context, focusNode, Function(List<EventModel>?) onApply) {
+    context,
```
[the remark this answers is not in the document]
Comment (asogwachiuba) [reply lib/core/constants/methods.dart:675]: Yeah true didnt even occur to me. Done

File: lib/features/events/views/eventlist/event_list_view.dart [hunk 6]
```diff
@@ -44,7 +44,7 @@ class EventListViewState extends State<EventListView> {
 
     return ViewModelBuilder<EventListViewModel>.reactive(
         viewModelBuilder: () => EventListViewModel(),
-        onModelReady: (viewModel) {
+        onViewModelReady: (viewModel) {
```
[the remark this answers is not in the document]
Comment (asogwachiuba) [reply lib/features/events/views/eventlist/event_list_view.dart:47]: Previous implementation when i was still new to the architecture. Corrected.

File: lib/features/events/views/explore/explore_events_view.dart (hunk 1 above)
Comment (asogwachiuba) [reply lib/features/events/views/explore/explore_events_view.dart:57]: Yes it was done to test, thought i changed it. Fixed.

File: lib/features/places/views/explore_places/explore_places_view.dart (hunk 4 above)
Comment (asogwachiuba) [reply lib/features/places/views/explore_places/explore_places_view.dart:18]: I deleted commented code in the file as I could not see this particular commented line.
